packages/node-opentelemetry/src/sentry.ts

```diff
@@ -157,28 +157,50 @@ const startOtelSpanFromSentryEvent = (event: Event, hint: EventHint) => {
   }
 };
 
-const registerBeforeSendEvent = (event: Event, hint: EventHint) => {
-  hdx('Received event at beforeSendEvent hook');
-  if (isSentryEventAnException(event)) {
-    startOtelSpanFromSentryEvent(event, hint);
+const registerEventProcessor = (event: Event, hint: EventHint) => {
+  try {
+    hdx('Received event');
+    if (isSentryEventAnException(event)) {
+      startOtelSpanFromSentryEvent(event, hint);
+    }
+  } catch (e) {
+    hdx(`Error processing event: ${e}`);
   }
+  // WARNING: always return the event
+  return event;
 };
 
 export const initSDK = async () => {
   try {
     const Sentry = await import('@sentry/node');
-    if (!Sentry.isInitialized()) {
-      hdx('Sentry SDK not initialized');
-      return;
-    }
-    const client = Sentry.getClient();
+    hdx(`Detected Sentry installed with SDK version: ${Sentry.SDK_VERSION}`);
+    const client = Sentry.getCurrentHub()?.getClient();
     if (!client) {
       hdx('Sentry client not found');
+    }
+    // TODO: initialize Sentry SDK ??
```

Comment:
Ahh missed this TODO - so for this release are we just assuming an initialized SDK and just not handling for non-existent and pushing that to a later release?

Reply:
correct, we are not going to init the sdk in the current version